src/ActionModal/components/ActionModal.vue

```diff
@@ -710,13 +709,15 @@ export default {
       // sending to ga
       this.sendEvent(
         {
-          network: this.network,
-          address: this.address
+          network: this.network.id,
+          address: this.session.address
         },
         "Action",
+        "Modal",
         this.featureFlag,
-        "",
-        this.amount
+        this.featureFlag == "claim_rewards"
+          ? this.rewards[0].amount
```

Comment:
this will not work with multi denom rewards 🤷‍♂

Reply:
We will fix it when we will have multi denoms)

Reply:
we already have multi denom rewards

Reply:
> we already have multi denom rewards

only on e-Money which is a testnet

Reply:
but we want to bring to mainnet asap to finish the milestone?!